src/wren_core.c

```diff
@@ -1009,6 +1009,36 @@ DEF_NATIVE(num_bitwiseOr)
   RETURN_NUM(left | right);
 }
 
+DEF_NATIVE(num_bitwiseXor)
+{
+  if (!validateNum(vm, args, 1, "Right operand")) return PRIM_ERROR;
+
+  // Bitwise operators always work on 32-bit unsigned ints.
+  uint32_t left = (uint32_t)AS_NUM(args[0]);
+  uint32_t right = (uint32_t)AS_NUM(args[1]);
+  RETURN_NUM(left ^ right);
+}
+
+DEF_NATIVE(num_bitwiseLsh)
```

Comment:
Nit, but do you mind calling this `num_bitwiseLeftShift` (and likewise for right)?